docs/Async.md

```diff
@@ -1,7 +1,9 @@
 # Asynchronous Actors for Message Bus
 
-Mockintosh offers "Mock Actor" approach for using with asynchronous message bus technologies, such as Kafka. "Mock
-Actor" approach requires you to provide deployed message bus instance, and configure valid address for it inside
+Mockintosh offers "Mock Actor" approach for using with asynchronous message bus technologies, such as
```

Comment:
I would explicitly mention RabbitMQ here, to help people understand that their async technology is supported.